### ISSUE TITLE
Fast vertical scrollbar blink on some pages

### ISSUE THREAD

**Issue by bcroq** [keepassxreboot/keepassxc-browser#952]
On some pages, the vertical scrollbar rapidly appears and disappears (of course, only when the keepassxc-browser extension is enabled).

It happens only once, when the page is loaded.

## Expected Behavior

The vertical scrollbar should not appear.

## Current Behavior

The vertical scrollbar rapidly appears and disappears when the page is loaded.

It happens when keepassxc.svg is *loaded*:
![Capture d’écran de 2020-07-21 16-58-30](https://user-images.githubusercontent.com/130780/88070868-769e8800-cb73-11ea-97a2-03065f6be690.png)

## Possible Solution

It seems to happen on pages that use body and first div with style "height:100%" but I don't know how to fix this issue.

## Steps to Reproduce

1. go to https://projects.invisionapp.com/d/main#/projects/prototypes/123456
2. click the "SIGN IN" button without filling the inputs
3. click the "SIGN IN" button without filling the inputs
4. ...

## Debug info

KeePassXC - 2.6.0
KeePassXC-Browser - 1.6.6
Operating system: Linux
Browser: Chrome and Firefox

**Comment by varjolintu** [comment]
Possibly related to this: https://github.com/keepassxreboot/keepassxc-browser/issues/896

**Comment by bcroq** [comment]
> Possibly related to this: #896

Exactly!

Duplicate of #896
